src/libs/ReportUtils.ts

```diff
@@ -4637,6 +4638,7 @@ function buildOptimisticChatReport(
     parentReportID = '',
     description = '',
     avatarUrl = '',
+    avatarFileName = '',
```

Comment:
After adding this new param we need to check all calls to `buildOptimisticChatReport` the 15th param should be `avatarFileName`. This is not the case in `buildOptimisticWorkspaceChats`. (Plaese double check all calls)

Reply:
@s77rt just checked this. This is fine at all the places

Reply:
Also conflicts are also resolved

Reply:
This is not fixed yet

Reply:
What is not fixed? I have verified in all places where`buildOptimisticChatReport ` is used.

Reply:
In `buildOptimisticWorkspaceChats` we call `buildOptimisticWorkspaceChats` and the 15th param is `expenseReportId` but that param is for `avatarFileName` and `expenseReportId` should be the 16th

Reply:
@s77rt  Sorry, for the miss, please have a look.